.eslintrc.cjs

```diff
@@ -43,7 +45,17 @@
   },
   "env": {"es6": true, "node": true, "mocha": true},
   "globals": {},
-  "plugins": ["json", "mocha", "@blackflux/rules"],
-  "parser": "@babel/eslint-parser",
-  "parserOptions": {"requireConfigFile": false}
-}
+  "plugins": ['@typescript-eslint', "json", "markdown", "mocha", "@blackflux/rules"],
+  "parser": "@typescript-eslint/parser",
+  parserOptions: {
```

Comment:
A small thing, but it'd be good to be consistent across the file in the usage of quotes. The previous file used double quotes everywhere because it was JSON, but now it's not necessary. I'd recommend keeping one standard and following it.

Reply:
Also, I don't actually see a point in migrating away from JSON file. Because there is no JS logic in the file anyway.
And apart from that, there are still few references to the old JSON file, you can find them in the code if you search `.eslintrc.json`

Reply:
Absolutely agreed on quote consistency. I was just playing around. Ironically eslint would have enforced this, but it's not working correctly yet =)

I tried using a json file, but then the `"@typescript-eslint"` plugin is not picked up correctly and I'm getting the error
`Cannot find package 'eslint-plugin-@typescript-eslint' imported from ...`

Reply:
Actually let me see if I can fix that...

Reply:
Still getting the same error now: 
```

Reply:
Still getting the same error now

```
.../object-treeify/tsconfig.eslint.json
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
```
And confused as to why. It's easy to reproduce by just running `yarn it`

Reply:
I think I have more work to do here. Stay tuned.................

Reply:
I'll try to check it on my side today also

Reply:
`yarn it` you say? Look at my output of this command, no errors:

<img width="897" alt="Screenshot 2022-09-14 at 11 28 35" src="https://user-images.githubusercontent.com/10692413/190117366-842c509d-fc9f-48f7-8ff3-ae7735b1f29e.png">

Do you have another output or you changed something meanwhile?